gcloud/datastore/test_query.py

```diff
@@ -32,35 +26,53 @@ def _getTargetClass(self):
     def _makeOne(self, *args, **kw):
         return self._getTargetClass()(*args, **kw)
 
-    def test_ctor_defaults_wo_implicit_dataset_id(self):
-        self.assertRaises(ValueError, self._makeOne)
+    def _makeClient(self, connection=None):
+        from gcloud.datastore.test_batch import _Client
```

Comment:
Ditto here on just using the original `_Client`. Also doesn't this contradict your manifesto? (I know we are doing it elsewhere, but that doesn't mean we should be.)